src/core/lang.mli

```diff
@@ -220,10 +197,8 @@ val error_t : t
 val source_t : ?methods:bool -> t -> t
 val of_source_t : t -> t
 val format_t : t -> t
-val kind_t : Frame.kind -> t
-val kind_none_t : t
-val frame_t : t Frame.Fields.t -> t
-val of_frame_t : t -> t Frame.Fields.t
+val frame_t : t -> t Frame.Fields.t -> t
```

Comment:
Please add detailed comment for this one. In particular, explain that the first `t` should either be a variable or `unit_t`. It might be more natural here and in related places to have the first argument a `t` option so that we can use `None` instead of `unit_t` for closed records?

Reply:
I will add a comment. I decided to leave it explicit to make the code more readable. When optional, you always have to wonder what is the default, universal or unit? Or something else (internal constraint for instance). Ideally, I'd like see an operator type and be able to figure out the whole deal:
```ocaml
let return_t =
  Lang.frame_t
    (Lang.univ_t ())
    (Frame.fields ~audio:(Format_type.audio ()) ())
in
```
Vs.:
```ocaml
let return_t =
  Lang.frame_t
    Lang.unit_t
    (Frame.fields ~audio:(Format_type.audio ()) ())
in
```
This was a common mistake when proof reading inputs such as `input.alsa` type (where it should be `unit_t`)